<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMemoryViewClass</class>
 <widget class="QMainWindow" name="TestMemoryViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestMemoryView</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>35</y>
      <width>86</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开内存浏览器</string>
    </property>
   </widget>
   <widget class="WinInfo" name="label_WIC">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="TestMemoryView.qrc">:/MyWidgets/WinInfoWidgets/Resources/drag.ico</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>95</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WinInfo</class>
   <extends>QLabel</extends>
   <header>WinInfoWidgets\WinInfo.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TestMemoryView.qrc"/>
 </resources>
 <connections/>
</ui>
